src/vector/jitsi/index.ts

```diff
@@ -20,7 +20,6 @@ require("./index.scss");
 import * as qs from 'querystring';
 import { Capability, WidgetApi } from "matrix-react-sdk/src/widgets/WidgetApi";
 import SdkConfig from "matrix-react-sdk/src/SdkConfig";
-import { loadConfig, preparePlatform } from "../initial-load";
 
 // Dev note: we use raw JS without many dependencies to reduce bundle size.
```

Comment:
Can you file a future backlog issue for the bundle size? (It seems the Jitsi bundle is quite large currently.)

Reply:
https://github.com/vector-im/riot-web/issues/12839